src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
```

Comment:
З огляду на оптимізацію було б краще використати такий підхід, оскільки метод find також працює як перебір массиву О(n) а метод has працює як звернення по ключу зі складністю О(1)
```suggestion
const mothersNameSet = new Set(people.filter((p) => p.mother)
    .map((p) => p.mother));
    
  const women = withChildren
```